### ISSUE TITLE
travis broke: cannot find conda

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/GridDataFormats#23]
No idea what's wrong: see eg https://travis-ci.org/MDAnalysis/GridDataFormats/jobs/94133691

```
0.00s$ source ~/virtualenv/python3.4/bin/activate
$ python --version
Python 3.4.2
$ pip --version
pip 6.0.7 from /home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages (python 3.4)
before_install.1
0.21s$ wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
--2015-12-01 10:33:01--  http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh
Resolving repo.continuum.io (repo.continuum.io)... 23.23.112.241, 54.83.3.29, 75.101.148.13, ...
Connecting to repo.continuum.io (repo.continuum.io)|23.23.112.241|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 23460669 (22M) [application/octet-stream]
Saving to: `miniconda.sh'
100%[======================================>] 23,460,669   121M/s   in 0.2s    
2015-12-01 10:33:02 (121 MB/s) - `miniconda.sh' saved [23460669/23460669]
before_install.2
0.00s$ chmod +x miniconda.sh
before_install.3
3.80s$ ./miniconda.sh -b
PREFIX=/home/travis/miniconda2
installing: python-2.7.10-2 ...
installing: conda-env-2.4.4-py27_0 ...
installing: openssl-1.0.2d-0 ...
installing: pycosat-0.6.1-py27_0 ...
installing: pyyaml-3.11-py27_1 ...
installing: readline-6.2-2 ...
installing: requests-2.8.1-py27_0 ...
installing: sqlite-3.8.4.1-1 ...
installing: tk-8.5.18-0 ...
installing: yaml-0.1.6-0 ...
installing: zlib-1.2.8-0 ...
installing: conda-3.18.3-py27_0 ...
installing: pycrypto-2.6.1-py27_0 ...
installing: _cache-0.0-py27_x0 ...
Python 2.7.10 :: Continuum Analytics, Inc.
creating default environment...
installation finished.
before_install.4
0.00s$ export PATH=/home/travis/miniconda/bin:$PATH
0.00s$ conda update --yes conda
/home/travis/build.sh: line 45: conda: command not found
The command "conda update --yes conda" failed and exited with 127 during .

```

Isn't it weird that for a Python 3.4 job, I see *Python 2.7.10 :: Continuum Analytics, Inc.*??

**Comment by richardjgowers** [comment]
I got this yesterday on MDA too, there's a restart job button on the top right that usually fixes stuff like this,

**Comment by richardjgowers** [comment]
Ok so that didn't work for me.

https://github.com/MDAnalysis/mdanalysis/commit/270c1cf25663c02325005a16afa3d8958aef0efc

We did have to change this a little while ago because conda installed into a different location than previously, but it looks like this step is passing?

**Comment by orbeckst** [comment]
5cb6f57 fixed it; the failure is now due to #21.

**Comment by orbeckst** [comment]
@richardjgowers, I attributed the commit to you because it was exactly your MDAnalysis/mdanalysis#520 – many thanks!
